src/Semantic/Api/Symbols.hs

```diff
@@ -78,15 +88,15 @@ parseSymbols blobs = do
           & P.language .~ (bridging # blobLanguage')
           & P.symbols .~ mempty
           & P.errors .~ [defMessage & P.error .~ pack e]
-          & P.blobOid .~ blobOid
+          & P.blobOid .~ mempty
```

Comment:
This is interesting.

1. What would be involved in updating the wire types to not include this field?

2. The wire types come with defaults for each field; does that make this equivalent to omitting the field?

3. If not, is the behaviour on the other end equivalent between this and omitting the field?

Reply:
1. We would need to track down all consumers of this API and make sure that they’re not using the `blobOid` field (which I don’t think they are).
2. Yes, the empty string is the default value for a `Text` field, so the behavior is preserved.
3. see above

Reply:
Given that, should we omit the fields?

Reply:
I wouldn’t object, but I would do it in a separate PR, since that touches semanticd functionality as well.

Reply:
Oh, I meant omit _setting_ the fields.

Reply:
Oh, I see. Yeah, roger that.